build/ci/unit-tests.yml

```diff
@@ -61,3 +61,21 @@ jobs:
     pool: 
       name: NetCorePublic-Pool
       queue: BuildPool.Windows.Amd64.VS2019.Pre.ES.Open
+
+- job: RichNav
```

Comment:
What is this job sharing from unit-tests? It seems like this should be separate pipeline.

Reply:
Yes, I'll move it to its own. As of now I was just testing if it would work - doesn't seem to be the case